tfhe/src/c_api/high_level_api/integers.rs

```diff
@@ -183,6 +183,16 @@ macro_rules! impl_operations_for_integer_type {
                     *result = Box::into_raw(Box::new($name(r)));
                 })
             }
+
+             // map cmux to if_then_else
+             pub unsafe extern "C" fn [<$name:snake _cmux>](
+                condition_ct: *const $name,
+                then_ct: *const $name,
+                else_ct: *const $name,
+                result: *mut *mut $name,
+            ) -> c_int {
+                [<$name:snake _if_then_else>](condition_ct, then_ct, else_ct, result)
```

Comment:
call cmux directly ?

Reply:
We could but then I would have to do 
```
 let condition_ct = &$crate::c_api::utils::get_ref_checked(condition_ct).unwrap().0;
                    let then_ct = &$crate::c_api::utils::get_ref_checked(then_ct).unwrap().0;
                    let else_ct = &$crate::c_api::utils::get_ref_checked(else_ct).unwrap().0;

                    let r = condition_ct.cmux(then_ct, else_ct);

                    *result = Box::into_raw(Box::new($name(r)));
```
so calling CAPI's if_then_else was quicker

Reply:
ok then all good